packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -325,6 +311,63 @@ export class TransactionPool {
 		);
 	}
 
+	private async processVerifiedTransactions(): Promise<CheckTransactionsResponse | void> {
```

Comment:
would it be possible to return the same type? maybe return something like `{ failed: [], passed: [] }`?